misc/ci/macos-aarch64-build-release.ini

```diff
@@ -0,0 +1,51 @@
+[constants]
+# apple silicon only supports MacOS 11.0, so just use that
+macos_min = '11.1'
+cflags = ['-mmacosx-version-min='+macos_min, '-arch', 'arm64']
+ldflags = cflags
+
+[host_machine]
+cpu_family = 'aarch64'
+cpu = 'aarch64'
```

Comment:
Not that it really matters, but if we're passing that `-mcpu` flag, might as well set `cpu` to the same value. `cpu_family` must still be `aarch64`, though (`cpu_family` is the broad architecture/instruction set, `cpu` is the specific microarch/cpu).